packages/manager/modules/vps/src/upscale/price/price.class.js

```diff
@@ -0,0 +1,12 @@
+export default class Price {
```

Comment:
Does it make sense to add it to @ovh-ux/manager-models module instead ? Looks like this could be used elsewhere

Reply:
I can't see where this is instanciated in the components. Is this class useful ?

Reply:
@marie-j yes indeaed it should be there, didn't know this existed

Reply:
@JeremyDec for documentation purpose, so yes it has the most important purpose 🙂

Reply:
Hmm even for documentation, it is weird as you suppose that the price used in (where?) has the same representation as this class, without any explicit instance.

Reply:
JavaScript is not statistically typed and I don't want to enforce the user the create an instance of Price each time they use the component, it would prevent them from using it in pure HTML

It's specified in the documentation that the type is Price

Reply:
@FredericEspiau Why not just add the documentation part where the object should be manipulated ? Given the huge code we already have to maintain, it looks to me like adding unused code is not a good idea